packaging/docker/pom.xml

```diff
@@ -34,7 +34,7 @@
         </dependency>
         <dependency>
             <groupId>org.alfresco.integrations</groupId>
-            <artifactId>alfresco-googledocs-share-community</artifactId>
+            <artifactId>alfresco-googledocs-share</artifactId>
```

Comment:
Why do you need to change this ?

Reply:
I was looking to  https://artifacts.alfresco.com/nexus/content/groups/internal/org/alfresco/integrations/alfresco-googledocs-share/3.1.0.1/ and thought the classifier was removed :). Reverted it. Thanks